src/mission-planner/cfg/mission_planner.cfg

```diff
@@ -0,0 +1,17 @@
+#test config file for the mission planner
+
+PACKAGE='mission-planner'
+import roslib; roslib.load_manifest(PACKAGE)
+
+from dynamic_reconfigure.parameter_generator import *
+
+gen = ParameterGenerator()
+
+#examples of what kind of things we could store as params.
+
+gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)
```

Comment:
This config file has test parameters and needs to be updated with the parameters we need for the bot. The other thing we can do with dynamic reconfigure is create parameter groups so we may be able to make a parameter group for each node.